crates/zksync/core/src/vm/inspect.rs

```diff
@@ -446,16 +475,73 @@ fn inspect_inner<S: ReadStorage>(
             bytecode_to_factory_dep(b.original.clone())
                 .expect("failed converting bytecode to factory dep")
         })
-        .collect();
+        .collect::<HashMap<U256, Vec<U256>>>();
     let modified_storage = if is_static {
         Default::default()
     } else {
         storage.borrow().modified_storage_keys().clone()
     };
 
-    InnerZkVmResult { tx_result, bytecodes, modified_storage, call_traces }
+    // patch CREATE traces.
+    for call in call_traces.iter_mut() {
+        call_traces_patch_create(&deployed_bytecode_hashes, &bytecodes, storage.clone(), call);
```

Comment:
@Jrigada wasn't there something we found with `vm.get_last_tx_compressed_bytecodes()` that resulted in it not having all recent factory deps or something?

Reply:
Anyhow, it should either be present in the returned bytecodes or the `StorageHandle`, since we extend it with `persisted_factory_deps` before each batch.

Reply:
If we are extending it with persisted_factory_deps then that's the fix to the issue I was mentioning